spec/factories/incidental_type.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :incidental_type do
+    sequence(:name) { |n| "name #{n}" }
```

Comment:
It's nice for the factory to echo the model validations - so you should use a `sequence` when you have a uniqueness validation, which it doesn't look like is the case here. A static string should be fine - or implement a uniqueness validation.